src/components/DatePicker/_DatePicker.scss

```diff
@@ -5,14 +5,22 @@ fieldset {
 
   &.datePicker {
     @include fieldset-reset;
+    margin-bottom: var(--space-m);
   }
 }
 
 .datePicker {
+  margin-bottom: var(--space-m);
```

Comment:
Like above, this margin is not necessary, as we are expecting consuming apps to use `Form` to layout form fields.